lib/helper/Playwright.js

```diff
@@ -3122,7 +3122,19 @@ async function saveVideoForPage(page, name) {
 async function saveTraceForContext(context, name) {
   if (!context) return;
   if (!context.tracing) return;
-  const fileName = `${`${global.output_dir}${pathSeparator}trace${pathSeparator}${Date.now()}_${clearString(name)}`.slice(0, 245)}.zip`;
+  const fileName = `${`${global.output_dir}${pathSeparator}trace${pathSeparator}${makeId(20)}_${clearString(name)}`.slice(0, 245)}.zip`;
```

Comment:
is the fileName unique for one test or it probably should be unique within one test step too?

* If it should be unique per test, use `mocha current test id`

* If it should uniq  within one test use already installed CodeceptJS dependency - `uuid` tp generate really unique id